### ISSUE TITLE
Label Filter User Interface

### ISSUE THREAD

**Issue by erluko** [erluko/flowgridviz#52]
Make it possible for users to change how each square displays based on the labels present. e.g. by adjusting the opacity of labels and having the square's opacity be equal to the scaled sum of the opacities  of the labels.

Imported from Georgia Tech github:
```
{"created_at":"2019-04-18T22:37:05Z","updated_at":"2019-04-19T03:31:00Z","closed_at":"2019-04-19T03:30:44Z","html_url":"https://github.gatech.edu/ekobrin3/2019-Spring-CSE6242-flowgridviz/issues/52","id":78957}
```

**Comment by erluko** [comment]
Fixed in tag v0.16.0

Imported from Georgia Tech github:
```
{"created_at":"2019-04-19T03:30:44Z","updated_at":"2019-04-19T03:30:44Z","html_url":"https://github.gatech.edu/ekobrin3/2019-Spring-CSE6242-flowgridviz/issues/52#issuecomment-42188","id":42188}
```

**Comment by erluko** [comment]
Note: uses *means* rather than *sums*

Imported from Georgia Tech github:
```
{"created_at":"2019-04-19T03:31:00Z","updated_at":"2019-04-19T03:31:00Z","html_url":"https://github.gatech.edu/ekobrin3/2019-Spring-CSE6242-flowgridviz/issues/52#issuecomment-42189","id":42189}
```
